c/tests/test_tables.c

```diff
@@ -2851,6 +2851,147 @@ test_link_ancestors_multiple_to_single_tree(void)
     tsk_treeseq_free(&ts);
 }
 
+static void
+test_ibd_finder(void)
+{
+    int ret;
+    // int j;
+    tsk_treeseq_t ts;
+    tsk_table_collection_t tables;
+    tsk_id_t samples[] = { 0, 1 };
+    tsk_ibd_finder_t ibd_finder;
+
+    tsk_treeseq_from_text(&ts, 1, single_tree_ex_nodes, single_tree_ex_edges, NULL, NULL,
+        NULL, NULL, NULL, 0);
+    ret = tsk_treeseq_copy_tables(&ts, &tables, 0);
+    CU_ASSERT_EQUAL_FATAL(ret, 0);
+
+    ret = tsk_ibd_finder_init_and_run(&ibd_finder, &tables, 1.0, samples, 2, 0.0, 0.0);
+    CU_ASSERT_EQUAL_FATAL(ret, 0);
+
```

Comment:
We're not really testing anything here - can we check the known result for a couple of sample pairs?